### DISCUSSION TITLE
Issue while installing SO Version: 2.3.80 (fleet and mysql missing)

### DISCUSSION THREAD

**Discussion by tbaror** [Security-Onion-Solutions/securityonion#5788] [Unsupported Versions]
Hello,
I did network installation on Ubuntu 18.04 latest update , after installation i did `so-status` i saw that fleet and mysql are status missing.
Next i did `so-docker-refresh` and started fleet and mysql , they show status running, but when i get into fleet  i get `404 page not found`
attached  sosetup.log and fleet logs   

Please advice
Thank you
[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/7293578/sosetup.log)

[fleet-setup.log](https://github.com/Security-Onion-Solutions/securityonion/files/7293557/fleet-setup.log)

**Comment by dougburks** [comment]
Do you have a firewall or some other device that may be interfering with your downloads?

When you ran `so-docker-refresh`, did it download anything new or did each container image just say `Layer already exists`?

Are you able to duplicate this issue on a fresh installation?

**Reply by tbaror** [reply]
Hello Doug,
Yes i have pfsense with squid and clam av activate on it , can it cause issues for that particular install? 
I did the fresh install several times and, always mysql and fleet is with issue 
Thanks

**Reply by dougburks** [reply]
Please try bypassing these features or try a different network and see if you get different results.
